packages/frint-store/src/createStore.spec.js

```diff
@@ -120,7 +120,7 @@ describe('frint-store › createStore', function () {
     const actions = [];
     const Store = createStore({
       enableLogger: false,
-      thunkArgument: { foo: 'bar' },
+      deps: { foo: 'bar' },
```

Comment:
I think for consistency it's better to rename all occurrences of `thunkArgument` in this test to `deps`

Reply:
that's what's done here. the `thunkArgument` line is in red :)